lib/slimkeyfy/console/translate.rb

```diff
@@ -98,4 +98,32 @@ def translations_are_invalid?(translations)
   def comment_tag
     @transformer.slim? ? "//" : "#"
   end
+
+  def join_multiline( strings_array )
+    result = []
+    joining_str = ''
+    indent_length = 0
+    long_str_start = /^[ ]+\|/
+    long_str_indent = /^[ ]+/
+    strings_array.each do |str|
+      if joining_str.empty?
+        if str[long_str_start]
+          joining_str = str
+          indent_length = str[long_str_start].length
+        else
+          result << str
+        end
+      #multiline string continues
+      elsif str[long_str_indent] && str[long_str_indent].length.to_i >= indent_length
+        joining_str << str.gsub( long_str_indent, ' ' )
+      #multiline string ends
+      else
+        result << joining_str
+        joining_str = ''
+        indent_length = 0
+        redo
```

Comment:
also given specs will not work cause it's not the word level, but level for SlimKeyfy::Console::Translate when it's importing and splittinng file